examples/dataflow-flex-python/df-sample-xml-to-bq/xmltobq_test.py

```diff
@@ -0,0 +1,70 @@
+import glob
+import json
+import logging
+import os
+
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.test_utils import TempDir
+import pytest
+
+import xmltobq
+
+SAMPLE_XML = """<?xml version="1.0" encoding="utf-8"?> 
```

Comment:
Same here, missing source attribution.
Also, you can read this in from sample-data/data.xml right?

Reply:
Downloaded relevant file and Reading it from sample-data now.